### PR TITLE
Adding RHEL 9 support for oc

### PR DESCRIPTION
**Pull request by bergerhoffer**
Version(s):
4.11

Issue:
https://issues.redhat.com/browse/OSDOCS-3800

Link to docs preview:
(single page) http://file.rdu.redhat.com/~ahoffer/2022/getting-started-cli.html#cli-installing-cli-rpm_cli-developer-commands

Additional information:

### CODE REVIEW

File: modules/cli-installing-cli-rpm.adoc [hunk 1]
```diff
@@ -43,6 +43,15 @@ For {op-system-base-full}, you can install the OpenShift CLI (`oc`) as an RPM if
 ----
 
 . Enable the repositories required by {product-title} {product-version}.
+
+** For Red Hat Enterprise Linux 9:
++
+[source,terminal]
+----
+# subscription-manager repos --enable="rhocp-4.11-for-rhel-9-x86_64-rpms"
```
Comment (bergerhoffer) [inline modules/cli-installing-cli-rpm.adoc:51]: @zhouying7780 If we are going to call oc supported with RHEL 9 for 4.11 (see question here: https://issues.redhat.com/browse/OSDOCS-3800?focusedCommentId=20558750&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-20558750) can you test and see what the correct repo is for installing via RPM for oc on RHEL 9?

**Review by soltysh — APPROVED** [review]
/lgtm

File: modules/cli-installing-cli-rpm.adoc (hunk 1 above)
Comment (zhouying7780) [reply modules/cli-installing-cli-rpm.adoc:51]: after install the bash-completion package on RHEL9 , I could install the oc client by yum repo:
cat /etc/yum.repos.d/openshift.repo
[ose-v4-client]
name=ose-v4-client
baseurl=http://download.eng.bos.redhat.com/rcm-guest/puddles/RHAOS/plashets/4.11/building/x86_64/os/
enabled=1
gpgcheck=0
metadata_expire=1

**Comment by bergerhoffer** [comment]
> This needs to be reverted, as such a repo does not exist for 4.11.

PR to revert/clarify no RPM support: #47923

**Pull request merged**